public/client/TracyProfiler.hpp

```diff
@@ -484,6 +484,28 @@ class Profiler
         TracyLfqCommit;
     }
 
+    static tracy_force_inline void Blob( uint64_t encoding, const void* data, size_t size, int callstack )
+    {
+        assert( size < (std::numeric_limits<uint16_t>::max)() );
+#ifdef TRACY_ON_DEMAND
+        if( !GetProfiler().IsConnected() ) return;
+#endif
+        if( callstack != 0 )
+        {
+            tracy::GetProfiler().SendCallstack( callstack );
+        }
+
+        auto ptr = tracy_malloc( size );
+        memcpy( ptr, data, size );
+
+        TracyQueuePrepare( callstack == 0 ? QueueType::Blob : QueueType::BlobCallstack );
+        MemWrite( &item->blobData.time, GetTime() );
```

Comment:
Time value needs to be send as a delta over the wire in Dequeue().

Reply:
This implementation is taken from the Message...() functions. As far as my understanding goes, it is converted in TracyWorker using the TscTime() function.

Reply:
The missing processing step is done in Deqeue(). Look for refCtx for examples of this.